WordPress/Tests/WP/GlobalVariablesOverrideUnitTest.1.inc

```diff
@@ -311,6 +311,14 @@ list(
 	get($year, $day) => &$not_a_wp_global[$year]
 ] = $array;
 
-// Live coding/parse error.
-// This has to be the last test in the file!
-list( $tab, $tabs
+/*
+ * $GLOBALS with non-string key, unable to determine if it is a global variable override or not.
```

Comment:
Ahum... it's definitely a global variable override, we just don't know whether it is one we should flag.

```suggestion
 * $GLOBALS with non-string key, unable to determine which variable is being overridden.
```